target/linux/lantiq/xway/base-files/lib/preinit/05_set_preinit_iface_lantiq

```diff
@@ -0,0 +1,5 @@
+set_preinit_iface() {
+	ifname=eth0
```

Comment:
This file should also not be needed after this patch:
https://patchwork.ozlabs.org/project/openwrt/list/?series=249780

Could someone please try this.